nodes/number/float_math.py

```diff
@@ -39,6 +41,9 @@ class FloatMathNode(bpy.types.Node, AnimationNode):
     bl_label = "Math"
 
     def operationChanged(self, context):
+        for op in operationItems:
```

Comment:
please don't use 'op' here. instead write "for item in operationItems"
I know that I wrote 'op' in the execution code but that was to improve readability. Here that is not the case (imo)